pybossa/util.py

```diff
@@ -368,6 +368,11 @@ def datetime_filter(source, fmt):
     return source.strftime(fmt)
 
 
+def validate_ownership_id(ownership_id):
```

Comment:
Quick question: So the function just checks format of the id, but there is no check to see if the id actually exists in PWHO?

Reply:
Yes. We cannot access internal Bloomberg services from pybossa at this time, so the current approach is to check if the format of the identifier is correct.